marimo/_runtime/context/utils.py

```diff
@@ -57,3 +57,7 @@ def get_mode() -> Optional[RunMode]:
         return "test"
 
     return None
+
+
+def import_guard() -> bool:
```

Comment:
1. Can you document the cases in which you think it makes sense to have this return `False`?
2. If we're keeping this, can you add a `TODO` to the code where we load marimo notebooks to patch this function to return `False`? (Or just implement the patch, since it should be small.)